browser/main/modals/PreferencesModal/InfoTab.js

```diff
@@ -20,21 +20,29 @@ class InfoTab extends React.Component {
     }
   }
 
+  componentDidMount() {
+    const { autoUpdateEnabled, amaEnabled } = ConfigManager.get()
+
+    this.setState({ config: { autoUpdateEnabled, amaEnabled } })
+  }
+
   handleLinkClick(e) {
     shell.openExternal(e.currentTarget.href)
     e.preventDefault()
   }
 
   handleConfigChange(e) {
-    const newConfig = { amaEnabled: this.refs.amaEnabled.checked }
+    const newConfig = {
+      amaEnabled: this.refs.amaEnabled.checked,
+      autoUpdateEnabled: this.refs.autoUpdateEnabled.checked
+    }
 
     this.setState({ config: newConfig })
```

Comment:
This function will change the `amaEnabled` value when you update `autoUpdateEnabled` even when users haven't click save yet. Demonstration:
![Peek 2020-05-26 16-14](https://user-images.githubusercontent.com/12984316/82859861-048f3580-9f6c-11ea-8881-1a1bff831bf2.gif)

Reply:
Thanks for catching that!

I've fixed it

![boostnote](https://user-images.githubusercontent.com/28135630/82887330-c24fff00-9f3f-11ea-9554-238e26e0f8f7.gif)